posts/ja/2023-02-18-23.0.0.1_ja.adoc

```diff
@@ -0,0 +1,199 @@
+blog_description: Open Liberty 23.0.0.1 リリースでは、Liberty が EPLv1 から EPLv2 へと再ライセンスされました。 また、このリリースには様々なバグフィックスが含まれています。
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+blog-available-in-languages:
+- lang: en
+  path: /blog/2023/02/07/23.0.0.1.html
+---
+= Open Liberty 23.0.0.1 の EPLv2 への再ライセンス
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+オリジナル・ブログ: link:{url-prefix}/blog/2023/02/07/23.0.0.1.html[Relicensing to EPLv2 in Open Liberty 23.0.0.1]
+
+
+翻訳: link:{url-prefix}https://github.com/kaori-asa[浅田 かおり]  image:https://avatars0.githubusercontent.com/kaori-asa[width=3%,lign="left"]
+
+
+Open Liberty 23.0.0.1 リリースでは、Liberty が EPLv1 から EPLv2 へと再ライセンスされました。 また、このリリースには様々なバグフィックスが含まれています。
+
+In link:{url-about}[Open Liberty] 23.0.0.1:
+
+* <<eplv2, Open LibertyのライセンスはEPLv2になりました>>
+* <<bugs, 主なバグフィックス>>
+
+link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23001+label%3A%22release+bug%22[23.0.0.1].で修正されたバグの一覧を見ることができます。
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
+
+
+[#run]
+
+== 23.0.0.1を使用してアプリを実行する
+
+link:{url-prefix}/guides/maven-intro.html[Maven]を使う時には下記の設定を使ってください。
+
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>23.0.0.1</version>
+    <type>zip</type>
+</dependency>
+----
+
+link:{url-prefix}/guides/gradle-intro.html[Gradle]の場合は、下記の設定を使ってください。
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[23.0.0.1,)'
+}
+----
+
+Dockerを使っているときは、下記のイメージを使ってください。
+
+[source]
+----
+FROM open-liberty
+----
+
+または、Open Libertyの link:{url-prefix}/downloads/[ダウンロードページ]からダウンロードしてください。
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack_ja.svg[Ask a question on Stack Overflow, align="center"]
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/24069
+// Contact/Reviewer: ReeceNana,cbridgha
+// // // // // // // // 
+[#eplv2]
+== Open LibertyのライセンスはEPLv2になりました
+2022年12月7日、ink:{url-prefix}/blog/2022/12/07/relicense-eplv2-proposal.html[Open LibertyのEPLv2への再ライセンス提案]のブログ記事を公開し、Open LibertyのライセンスをEPLv1からEPLv2への変更についてコミュニティに意見を求めました。 この提案に対して否定的なフィードバックがなかったため、私たちはこの変更を進めました。結果として、Open Liberty 23.0.0.1はEPLv2ライセンスのもとでリリースされます。 EPLv1からEPLv2への移行の理由と影響についてのより詳しい情報は、link:{url-prefix}/blog/2022/12/07/relicense-eplv2-proposal.html[proposal blog記事]をご覧ください。
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+[#bugs]
+== 主なバグ修正
+
+以下のセクションでは、このリリースで修正したバグの一部について説明します。興味がある場合はlink:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23001+label%3A%22release+bug%22[full list of bugs fixed in 23.0.0.1]を参照してください。
```

Comment:
`full list of bugs fixed in 23.0.0.1` link is not translated

Reply:
Translated